sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/factors/variables/TargetV.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.sensitivity.factors.variables;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.powsybl.sensitivity.SensitivityVariable;
+
+import java.util.Objects;
+
+/**
+ * Sensitivity variable on a voltage target at a regulating terminal
+ *
+ * @author Anne Tilloy {@literal <anne.tilloy at rte-france.com>}
+ */
+public class TargetV extends SensitivityVariable {
+
+    @JsonProperty("variableId")
```

Comment:
I think it should be regulatingTerminalId here ?  We should add those new classes in the JSON tests.

Reply:
After a short negotiation with @sylvlecl, it seems that in a first approach we can rely on the equipement ID here.